src/game.zig

```diff
@@ -697,9 +747,18 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 		const right = Vec3d{-forward[1], forward[0], 0};
 		var movementDir: Vec3d = .{0, 0, 0};
 		var movementSpeed: f64 = 0;
+		
+		var crouch: bool = false;
```

Comment:
This variable is assigned once and used once, so why not just insert the `KeyBoard.key("crouch").pressed` into the current 1 use case of this variable?